Fix for package dependencies

ederc opened the pull request:
Should fix missing dependencies and removes log file from the repo.

a-kulkarn commented:
I've changed the master branch to totally remove the dependency on DynamicPolynomials and MultivariatePolynomials. Could you recheck the dependency issue?

ederc commented:
now i still get the warning for LinearAlgebra. shall i add it to the
dependencies?

On 06/24/19, a-kulkarn wrote:
> I've changed the master branch to totally remove the dependency on DynamicPolynomials and MultivariatePolynomials. Could you recheck the dependency issue?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/a-kulkarn/pAdicSolver/pull/2#issuecomment-504973995

-- 
gpg: http://hkps.pool.sks-keyservers.net/pks/lookup?search=0x6e0d034b959fde6a&fingerprint=on&op=index

a-kulkarn commented:
I've added the dependency and pushed it. It should work now.

Best,
Avi

On Tue, Jun 25, 2019 at 12:46 PM ederc <notifications@github.com> wrote:

> now i still get the warning for LinearAlgebra. shall i add it to the
> dependencies?
>
> On 06/24/19, a-kulkarn wrote:
> > I've changed the master branch to totally remove the dependency on
> DynamicPolynomials and MultivariatePolynomials. Could you recheck the
> dependency issue?
> >
> > --
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub:
> > https://github.com/a-kulkarn/pAdicSolver/pull/2#issuecomment-504973995
>
> --
> gpg:
> http://hkps.pool.sks-keyservers.net/pks/lookup?search=0x6e0d034b959fde6a&fingerprint=on&op=index
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/a-kulkarn/pAdicSolver/pull/2?email_source=notifications&email_token=AHEEZKKTPTX6WNIE7QX7OSLP4HZQDA5CNFSM4H2XDNGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODYP2M5Q#issuecomment-505390710>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHEEZKKJM3Y2EDM7WUAWMLDP4HZQDANCNFSM4H2XDNGA>
> .
>

ederc commented:
alright, works now.

On 06/25/19, a-kulkarn wrote:
> I've added the dependency and pushed it. It should work now.
> 
> Best,
> Avi
> 
> On Tue, Jun 25, 2019 at 12:46 PM ederc <notifications@github.com> wrote:
> 
> > now i still get the warning for LinearAlgebra. shall i add it to the
> > dependencies?
> >
> > On 06/24/19, a-kulkarn wrote:
> > > I've changed the master branch to totally remove the dependency on
> > DynamicPolynomials and MultivariatePolynomials. Could you recheck the
> > dependency issue?
> > >
> > > --
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly or view it on GitHub:
> > > https://github.com/a-kulkarn/pAdicSolver/pull/2#issuecomment-504973995
> >
> > --
> > gpg:
> > http://hkps.pool.sks-keyservers.net/pks/lookup?search=0x6e0d034b959fde6a&fingerprint=on&op=index
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/a-kulkarn/pAdicSolver/pull/2?email_source=notifications&email_token=AHEEZKKTPTX6WNIE7QX7OSLP4HZQDA5CNFSM4H2XDNGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODYP2M5Q#issuecomment-505390710>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AHEEZKKJM3Y2EDM7WUAWMLDP4HZQDANCNFSM4H2XDNGA>
> > .
> >
> 
> 
> -- 
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/a-kulkarn/pAdicSolver/pull/2#issuecomment-505398893

-- 
gpg: http://hkps.pool.sks-keyservers.net/pks/lookup?search=0x6e0d034b959fde6a&fingerprint=on&op=index
